src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -985,6 +1014,7 @@ class CreateVmModal extends React.Component {
                         storagePoolRefresh({ connectionName: storagePool.connectionName, objPath: storagePool.id });
                 }
             }, (exception) => {
+                console.info(`spawn 'vm creation' returned error: "${JSON.stringify(exception)}"`);
```

Comment:
Unrelated to this commit. Please make this either a `logDebug` (if the error already appears in the UI), or drop it if the debug log already contains it (which I'm rather sure of), or turn into a console.error().